lib/players/human.rb

```diff
@@ -0,0 +1,11 @@
+module Players
+
+  class Human < Player
+
+    def move(board)
+      input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  gets.strip
end
```